src/components/data/toolbar/DataDotMenu.js

```diff
@@ -88,7 +90,8 @@ function DataDotMenu(props) {
         : null;
     const deleteMiEnabled = !isSelectionEmpty && isOwner(selectedResources);
     const metadataMiEnabled = selected?.length === 1;
-
+    const requestDoiEnabled =
+        isOwner(selectedResources) && containsFolders(selectedResources);
```

Comment:
Also `selected?.length === 1`?

Reply:
I thought about that but then `Request DOI` menu item is inside `metadataMiEnabled` check so I didn't add that check 🤷🏽‍♂️

Reply:
Ah, right, I missed that. That will work then.